lib/src/main/java/com/scalar/admin/k8s/Pauser.java

```diff
@@ -42,6 +42,7 @@ public class Pauser {
   /**
    * @param namespace The namespace where the pods are deployed.
    * @param helmReleaseName The Helm release name used to deploy the pods.
+   * @throws PauserException when the default Kubernetes client fails to be set.
```

Comment:
There came a warning about exception declaration when we built the Java doc, so we fixed this as well.